app/(authenticated)/leads/page.tsx

```diff
@@ -1,9 +1,11 @@
+import LeadDetails from "@/components/ui/LeadDetails/lead-details";
 import LeadsList from "@/components/ui/LeadsList/LeadsList";
 
 const LeadsPage = () => {
   return (
     <div className=''>
       <LeadsList/>
+      {/* <LeadDetails/> */}
```

Comment:
If this code is not necessary then it is better to remove this line.

Reply:
Done